traitsui/qt4/text_editor.py

```diff
@@ -48,6 +46,13 @@ class SimpleEditor(Editor):
     #: Function used to evaluate textual user input:
     evaluate = evaluate_trait
 
+    # -- private trait definitions ------------------------------------------
+
+    #: A list of Qt signals connected.
+    #: First item in the tuple is the Qt signal. The second item is the event
+    #: handler.
+    _signals = List(Tuple(Any, Callable))
```

Comment:
Suggestion on the name - maybe it would be better to use `connections_to_remove` or something similar? Because (1) the list doesn't contain just signals, it contains signals and handlers; (2) there might be connections that shouldn't be disconnected in `dispose` method (e.g. to signal `destroyed`) and the name would make it clear that those connections shouldn't be appended to the list.

Reply:
Agreed with (1). Just wanted to clear, we still want this variable to be private right, i.e. with a preceding underscore?  If (2) does come up, then traitsui will have more flexibility to rename an internal private variable.

Reply:
I think it should be private